docs/5.0/application-access.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Application Access Guide
+description: How to set up and configure Teleport for Application access with SSO and RBAC
+---
+
+# Teleport Application Access
+
+## Introduction
+Application Access has been designed to provide secure access to internal dashboards and applications. Building on Teleports strong foundations of security and identity. You can now put applications onto the internet safely and securely.
```

Comment:
Application access provides access to internal dashboards and applications without using VPNs.